pbft.tla

```diff
@@ -591,8 +591,11 @@ GetDigest(ppms, sn) ==
 GenerateO(V,i,v) ==
     LET mins == Max0(UNION {{cp.n: cp \in vcm.c}: vcm \in V}) 
         ppms == UNION {{pp.preprepare: pp \in vcm.p}: vcm \in V}
-        maxs == Max0({ppm.n: ppm \in ppms}) IN
-    {[v |-> v, p |-> i, n |-> sn, d |-> GetDigest(ppms,sn)] : sn \in (mins+1)..maxs}
+        maxs == Max0({ppm.n: ppm \in ppms}) 
+        \* Apalache does not yet support integer ranges with non-constant bounds:
+        \* https://github.com/apalache-mc/apalache/blob/main/docs/src/apalache/known-issues.md#integer-ranges-with-non-constant-bounds
+        apa == { j \in Views : mins+1 <= j /\ j <= maxs} IN
```

Comment:
@heidihoward Could you please confirm that it is correct to quantify over `Views`?

Reply:
it should be quantified over `SeqNums` instead